scripts/Diffraction/isis_powder/osiris.py

```diff
@@ -179,6 +259,13 @@ def get_vanadium_path(self, run_details):
 
         return run_details.unsplined_vanadium_file_path
 
+    def get_vanadium_big_threshold(self):
```

Comment:
does this just provide a number, then if the value is greater than that number we ignore it?

Reply:
Ok, I will change it